src/sequence_statistics.py

```diff
@@ -0,0 +1,229 @@
+import requests
+import random
+import json
+
+
+# API_URL = 'http://0.0.0.0:5000'
+API_URL = 'http://127.0.0.1:5001'
+
+
+# -------------- Fonctions pour calculer les stats sur les syllabes ------------------------- #
+
+
+# Ici, pour un mot donné, on incrémente les statistiques de ses syllabes adjacentes
```

Comment:
I really appreciate you comment well the code. This is great practice. Even better if you do a pydoc (between 3 double quotes after the `def` line) to put them. Also even better if you can comment in English, it's the preferred language (if you can)